profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -60,28 +66,34 @@ private RfElementConfiguration() {
     	setMimoMode(MimoMode.twoByTwo);
     	setMaxNumClients(100);
     	setMulticastRate(MulticastRate.auto);
-//    	setAutoChannelSelection(false);
+    	setAutoChannelSelection(false);
     	setActiveScanSettings(ActiveScanSettings.createWithDefaults());
-//    	setManagementRate(ManagementRate.auto);
-//    	setRxCellSizeDb(AutoOrManualValue.createAutomaticInstance(DEFAULT_RX_CELL_SIZE_DB));
-//    	setProbeResponseThresholdDb(AutoOrManualValue.createAutomaticInstance(-90));
-//    	setClientDisconnectThresholdDb(AutoOrManualValue.createAutomaticInstance(-90));
-//    	setEirpTxPower(AutoOrManualValue.createAutomaticInstance(DEFAULT_EIRP_TX_POWER));
-//    	setBestApEnabled(null);
-//    	setNeighbouringListApConfig(NeighbouringAPListConfiguration.createDefault());
-//    	setPerimeterDetectionEnabled(true);
-//    	setChannelHopSettings(ChannelHopSettings.createWithDefaults());
+    	setManagementRate(ManagementRate.auto);
+    	setRxCellSizeDb(AutoOrManualValue.createAutomaticInstance(DEFAULT_RX_CELL_SIZE_DB));
+    	setProbeResponseThresholdDb(AutoOrManualValue.createAutomaticInstance(-90));
+    	setClientDisconnectThresholdDb(AutoOrManualValue.createAutomaticInstance(-90));
+    	setEirpTxPower(AutoOrManualValue.createAutomaticInstance(DEFAULT_EIRP_TX_POWER));
+    	setBestApEnabled(null);
+    	setNeighbouringListApConfig(NeighbouringAPListConfiguration.createDefault());
+    	setPerimeterDetectionEnabled(true);
+    	setChannelHopSettings(ChannelHopSettings.createWithDefaults());
     }
     
     public static RfElementConfiguration createWithDefaults(RadioType radioType) {
         RfElementConfiguration ret = new RfElementConfiguration();  
-//      ret.setBestApSettings(RadioBestApSettings.createWithDefaults(radioType));
+        ret.setBestApSettings(RadioBestApSettings.createWithDefaults(radioType));
         if (radioType == RadioType.is5GHz || radioType == RadioType.is5GHzL || radioType == RadioType.is5GHzU) {
```

Comment:
I suggest adding a new boolean method to RadioType to return true if it's param is any type of 5G. We'll have other cases which will need this kind of check eg boolean is5gFrequency(RadioType radioType). Someone else might have a better name.

Reply:
Maybe I should have done the reverse to simplify the logic, to check if it's 2.4G to set 20MHz, else set 80MHz. 

But if we are to support more radios in the future, all the else cases for RadioType will need to be refactored as well, not just here.